tests/forms_tests/widget_tests/test_multiwidget.py

```diff
@@ -17,6 +17,11 @@ def decompress(self, value):
         return ['', '']
 
 
+class MyMultiRenderWidget(MultiWidget):
```

Comment:
Is this test applicable to the patch anymore? It doesn't seem to me that overriding `attrs` with `render()` is related to what this patch fixes.